src/backend/distributed/sql/udfs/citus_internal_database_command/latest.sql

```diff
@@ -0,0 +1,10 @@
+--
+-- citus_internal_database_command creates a database according to the given command.
+
+CREATE OR REPLACE FUNCTION pg_catalog.citus_internal_database_command(command text)
+ RETURNS void
+ LANGUAGE C
+ STRICT
+AS 'MODULE_PATHNAME', $$citus_internal_database_command$$;
+COMMENT ON FUNCTION pg_catalog.citus_internal_database_command(text) IS
+ 'run a database command without transaction block restrictions';
```

Comment:
Let's remove STRICT and add a check into function def in C (PG_ENSURE_ARGNOTNULL) because we don't expect ourselves to call this function with NULL input and add a test to show that this throws an error on NULL input.

Let's also add VOLATILE to let Postgres know about this modifies database state.

```suggestion
CREATE OR REPLACE FUNCTION pg_catalog.citus_internal_database_command(command text)
 RETURNS void
 LANGUAGE C
 VOLATILE
AS 'MODULE_PATHNAME', $$citus_internal_database_command$$;
COMMENT ON FUNCTION pg_catalog.citus_internal_database_command(text) IS
 'run a database command without transaction block restrictions';
```